docs/sources/configure/options.md

```diff
@@ -939,6 +957,24 @@ the OpenTelemetry exporter will automatically add the `/v1/traces` path to the U
 addition, you can use either the `OTEL_EXPORTER_OTLP_TRACES_ENDPOINT` environment variable or the `environment` YAML
 property to use exactly the provided URL without any addition.
 
+| YAML               | Environment variable                  | Type            | Default                      |
```

Comment:
Is this wrongly duplicated or it's a different section?

Reply:
They are different sections. We can specify which instrumentations should be enabled in each of the OTel metrics, Prometheus metrics, Traces. The variable names are different, following the similar naming conventions as the other variables.